### DISCUSSION TITLE
How to keep music playing while navigating to different pages livewire way?

### DISCUSSION THREAD

**Discussion by amirandev** [livewire/livewire#7705] [1. Help] [answered]
Yesterday I found my old music website design which looks cool. In my web design I have a music player fixed on the bottom of the screen, similar to soundcloud, dizzer.. and etc.

I decided to build it with laravel and livewire but I got stucked.

**Why?**
Because code of the whole page is updating, including layout. By the reason the audio player is refreshing too.

**How can I keep the player playing while navigating?**
Damn... I've tried to use wire:ignore but it ignores my wish instead to keep the player unrefreshable.

**Question:**
It is possible to keep the player unrefreshable livewire way or I've to use JQuery that can do everything I wish,  again as we, web developers used to long ago?

**Comment by joshhanley** [comment, marked as answer]
@amirandev try this 🙂 https://livewire.laravel.com/docs/navigate#persisting-elements-across-page-visits

**Reply by amirandev** [reply]
Hell yeah, Thanks a lot. 

I've just put my player in `@presist *** @endpresist` and boom... 

```
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <title>{{ $title ?? 'Page Title' }}</title>
</head>
<body>
    @persist('player')
        <audio src="{{ asset('1.mp3') }}" controls></audio>
    @endpersist

    @livewire('header')

    {{ $slot }}

    @livewire('footer')
</body>
</html>

```
